modules/vstudio/vs2010_vcxproj.lua

```diff
@@ -696,13 +733,52 @@
 		end
 	}
 
+---
+-- ClCompile group
+---
+	m.categories.FxCompile = {
+		name	   = "FxCompile",
+		extensions = { ".hlsl" },
+		priority   = 3,
```

Comment:
Does this need to be priority 3?

Reply:
Not really.
I implemented it some time ago so I don't remember it that clearly but it might be that vs add this block just after the ClCompile block so I wanted to reproduce this.

I can update this to keep the previous priority order if you'd prefer.

Reply:
Fair enough, I see no reason to hold the PR up over this. What you have is fine.